ui/jobs/jobs.js

```diff
@@ -1108,6 +1108,17 @@ class Bars {
     ];
   }
 
+  uiDisplay(options) {
```

Comment:
What about instead of `uiDisplay(options)` instead this was `SetVisible(visible)` and you could call `SetVisible(true)` or `SetVisible(false)`? I think that would be more readable than passing `'none'` and `''` in two separate places.

Alternatively, you could have `UpdateUIVisibility()` and have it check `this.isPVPZone` internally rather than passing anything.